rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -637,11 +711,11 @@ s32 cellGemGetState(u32 gem_num, u32 flag, u64 time_parameter, vm::ptr<CellGemSt
 		map_to_ds3_input(gem_num, gem_state->pad.digitalbuttons, gem_state->pad.analog_T);
 		map_ext_to_ds3_input(gem_num, gem_state->ext);
 
-		gem_state->tracking_flags = CELL_GEM_TRACKING_FLAG_POSITION_TRACKED |
-									CELL_GEM_TRACKING_FLAG_VISIBLE;
+		gem_state->tracking_flags = CELL_GEM_TRACKING_FLAG_POSITION_TRACKED | CELL_GEM_TRACKING_FLAG_VISIBLE;
 		gem_state->timestamp = gem->timer.GetElapsedTimeInMicroSec();
 
 		gem_state->quat[3] = 1.0;
+		return CELL_OK;
```

Comment:
this is probably the reason for Top Darts to progress to the next screen xD

Reply:
yup lol